builtin/client/chatcommands.lua

```diff
@@ -2,6 +2,10 @@
 
 
 core.register_on_sending_chat_messages(function(message)
+	if message:sub(1,2) == ".." then
```

Comment:
That would just get messy if we add a way to change the command prefix later on and why would you want to do that anyway? Worldedit only does that because it's a clone of an existing minecraft mod.